templates/Yves/Plugin/Router/RouteProviderPlugin.php.twig

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace {{ namespace }};
+
+use Spryker\Yves\Router\Plugin\RouteProvider\AbstractRouteProviderPlugin;
+use Spryker\Yves\Router\Route\RouteCollection;
+
+class {{ classname }} extends AbstractRouteProviderPlugin
+{
+
```

Comment:
Redundant empty line. Please use code style sniffer for detecting these kinds o things.